ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -118,7 +120,13 @@ async def _wrapped_api_call(original_func, instance, args, kwargs):
     endpoint_name = deep_getattr(instance, "_endpoint._endpoint_prefix")
 
     service = pin.service if pin.service != "aws" else "{}.{}".format(pin.service, endpoint_name)
-    with pin.tracer.trace("{}.command".format(endpoint_name), service=service, span_type=SpanTypes.HTTP) as span:
+    with pin.tracer.trace(
+        schematize_cloud_api_operation(
```

Comment:
this is sub-par compared to what happens in botocore, but was discussed with the service name group.